src/games/strategy/triplea/Properties.java

```diff
@@ -677,5 +677,9 @@ public static boolean getControlAllCanalsBetweenTerritoriesToPass(final GameData
     return data.getProperties().get(CONTROL_ALL_CANALS_BETWEEN_TERRITORIES_TO_PASS, false);
   }
 
+  public static boolean getRollRocketsAfterSBR(final GameData data) {
+    return data.getProperties().get("Roll rocket damage after processing SBR", false);
```

Comment:
Why should this be a map property and not a function of rule set?  This is a very big deal adding a new map property.